src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php

```diff
@@ -47,6 +47,28 @@ public function register_routes() {
 				'schema' => array( $this, 'get_item_schema' ),
 			)
 		);
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/(?P<name>[\w-]+)',
```

Comment:
```suggestion
			'/' . $this->rest_base . '/(?P<stylesheet>[\w-]+)',
```

To match the property name in the schema.